Ports/openssl/package.sh

```diff
@@ -1,14 +1,40 @@
 #!/usr/bin/env -S bash ../.port_include.sh
-port=openssl
+port='openssl'
 branch='1.1.1'
-version="${branch}q"
-useconfigure=true
-configscript=Configure
-files="https://www.openssl.org/source/openssl-${version}.tar.gz openssl-${version}.tar.gz d7939ce614029cdff0b6c20f0e2e5703158a489a72b2507b8bd51bf8c8fd10ca"
-auth_type=sha256
+version="${branch}s"
+useconfigure='true'
+configscript='Configure'
+files="https://www.openssl.org/source/openssl-${version}.tar.gz openssl-${version}.tar.gz c5ac01e760ee6ff0dab61d6b2bbd30146724d063eb322180c6f18a6f74e4b6aa"
+auth_type='sha256'
+depends=(
+    'zlib'
+)
 
-depends=("zlib")
-configopts=("--prefix=/usr/local" "-DOPENSSL_SYS_SERENITY=1" "-DOPENSSL_USE_IPV6=0" "zlib" "threads" "no-tests" "no-asm" "serenity-generic")
+COMPILE_TESTS='true'
+
+configopts=(
+    "--prefix=/usr/local"
+    "--openssldir=/usr/local/ssl"
```

Comment:
What do you mean why? Why not something else? Or why include this at all?

Anyway, I've been experimenting with installing OpenSSL@1 in the recommendation location according to the OpenSSL documentation to get ready for the introduction of OpenSSL@3. This PR is still a draft.

Reply:
> What do you mean why? Why not something else? Or why include this at all?

Why we are setting this to a new, non-default path at all instead of leaving it at the old setting. Considering the followup commits, this does not appear to be something that other ports expect.

> This PR is still a draft.

Ah, whoops, that I missed. I thought I saw it in the review channel, which is the reason why I looked at the PR in the first place.